seqr/management/commands/add_project_tag_tests.py

```diff
@@ -17,11 +21,62 @@ class AddProjectTagTest(TestCase):
 
     def test_normal_command(self):
         out = StringIO()
-        call_command('add_project_tag', '--project={}'.format(TEST_ARGUMENTS["project"]),
-            '--name={}'.format(TEST_ARGUMENTS["name"]), '--order={}'.format(TEST_ARGUMENTS["order"]),
-            '--category={}'.format(TEST_ARGUMENTS["category"]),
-            '--description={}'.format(TEST_ARGUMENTS["description"]),
-            '--color={}'.format(TEST_ARGUMENTS["color"]), stdout = out)
-
-        expectOutput = out.getvalue()
-        self.assertIn('', expectOutput)
+        call_command('add_project_tag', '--project={}'.format(TAG_ARGUMENTS["project"]),
+            '--name={}'.format(TAG_ARGUMENTS["name"]), '--order={}'.format(TAG_ARGUMENTS["order"]),
+            '--category={}'.format(TAG_ARGUMENTS["category"]),
+            '--description={}'.format(TAG_ARGUMENTS["description"]),
+            '--color={}'.format(TAG_ARGUMENTS["color"]), stdout = out)
+
+        self.assertIn('', out.getvalue())
+
+        project = Project.objects.get(Q(name = TAG_ARGUMENTS["project"]))
+        variantTagType = VariantTagType.objects.get(name__iexact=TAG_ARGUMENTS["name"], project=project)
```

Comment:
theres no reason for `iexact` here, you can just do `name=`

Reply:
Okay. It was from a copy-paste. Remove it.